client/me/purchases/pre-cancellation-dialog/get-plan-cancellation-features.tsx

```diff
@@ -1,7 +1,41 @@
-export default function getPlanCancellationFeatures( productSlug: string | undefined ): string[] {
+import { getPlan, isMonthly } from '@automattic/calypso-products';
+
+export default function getPlanFeatures(
```

Comment:
I think the previous name `getPlanCancellationFeatures()` is much more precise, even it's a default export that the importing file can easily rename. Why would we want to rename this as `getPlanFeatures()` instead?

Reply:
I restored the getPlanCancellationFeatures function